pages/style-guide/prose.md

```diff
@@ -0,0 +1,111 @@
+<script>{
+    "title": "Prose Style Guide"
+}</script>
+
+## Formatting Conventions
+
+Articles about all content (sites, README, docs, etc) are authored with [GitHub Flavored Markdown](http://github.github.com/github-flavored-markdown/).
+
+### Article Header Metadata
+
+Each article should have the following header (see below as some metatags are optional):
+
+```
```

Comment:
@arthurvr Which part should be applied syntax highlighting to? The first four lines or from line 14 to line 22 (article header metadata)?